pkg/apis/crd/v1alpha1/types.go

```diff
@@ -143,30 +146,30 @@ type Destination struct {
 // IPHeader describes spec of an IPv4 header.
 type IPHeader struct {
 	// SrcIP is the source IP.
-	SrcIP string `json:"srcIP,omitempty"`
+	SrcIP string `json:"srcIP,omitempty" yaml:"srcIP,omitempty"`
```

Comment:
why the yaml tag is only added to some structs? Is there a difference?

Reply:
I added it only for structs will be printed by antctl. If you think I should add it for all, I can do that too.